sql/database.sql

```diff
@@ -1526,22 +1526,40 @@ CREATE  TABLE `erx_ttl_touch` (
 -----------------------------------------------------------
 
 --
--- Table structure for table `erx_drug_paid`
---
-
-DROP TABLE IF EXISTS `erx_drug_paid`;
-CREATE TABLE `erx_drug_paid` (
-  `drugid` int(11) NOT NULL AUTO_INCREMENT,
-  `drug_label_name` varchar(45) NOT NULL,
-  `ahfs_descr` varchar(45) NOT NULL,
-  `ndc` bigint(12) NOT NULL,
-  `price_per_unit` decimal(5,2) NOT NULL,
-  `avg_price` decimal(6,2) NOT NULL,
-  `avg_price_paid` int(6) NOT NULL,
-  `avg_savings` decimal(6,2) NOT NULL,
-  `avg_percent` decimal(6,2) NOT NULL,
-   PRIMARY KEY (`drugid`)
-   ) ENGINE=InnoDB;
```

Comment:
Are we dropping this table?
If so, then should also drop it in the sql upgrade script

Reply:
Yes, dropping the old table for the new one.  I did not drop the old one because some people have talked to me about modifying that table. But, I will add the drop command.

Reply:
A practice that is actively using Weno and does the upgrade will not know that the table has been dropped and replaced. The data should be imported for them during the upgrade process. 
Or should we provide an instruction to go to the Weno Admin and run the data import from there. Suggestions please.

Reply:
hi @juggernautsei ,
How were we importing the data in the prior erx_drug_paid table?
-brady

Reply:
https://github.com/openemr/openemr/blob/master/interface/weno/drugPaidInsert.php

I have been studying how this was constructed. The admin section is set up to alert the user that the user the table is empty. For those that upgrade, they will be contacting me to say that their drug data is missing. I will have to send an email to each saying that they need to go to the admin section and import the drug table.  There are 20k+ lines in the drug import.

Or, we can build a mechanism to install it for them if the drug table is populated on upgrade.

Reply:
hi @juggernautsei ,
First change the admin script to use your new table along with a new companion database insert file there for the new table. Then can definitely support auto upgrade with a custom upgrade function.To see how we do this, recommend checking out prior custom upgrade functions near top of this script: /home/brady2/git/openemr/library/sql_upgrade_fx.php